src/main/resources/assets/inventorytweaks/itemtree.xml

```diff
@@ -16,1181 +16,10191 @@
-                        <lingeringPotionWaterBreathingEx id="lingering_potion"
-                                                         data='{Potion:"minecraft:long_water_breathing"}'/>
-                    </waterBreahingLingeringPotion>
-                    <luckLingeringPotion>
-                        <lingeringPotionLuck id="lingering_potion" data='{Potion:"minecraft:luck"}'/>
-                    </luckLingeringPotion>
-                </positiveLingeringPotion>
-                <negativeLingeringPotion>
-                    <poisonLingeringPotion>
-                        <lingeringPotionPoison id="lingering_potion" data='{Potion:"minecraft:poison"}'/>
-                        <lingeringPotionPoisonEx id="lingering_potion" data='{Potion:"minecraft:long_poison"}'/>
-                        <lingeringPotionPoisonII id="lingering_potion" data='{Potion:"minecraft:strong_poison"}'/>
-                    </poisonLingeringPotion>
-                    <weaknessLingeringPotion>
-                        <lingeringPotionWeakness id="lingering_potion" data='{Potion:"minecraft:weakness"}'/>
-                        <lingeringPotionWeaknessEx id="lingering_potion" data='{Potion:"minecraft:long_weakness"}'/>
-                    </weaknessLingeringPotion>
-                    <slownessLingeringPotion>
-                        <lingeringPotionSlowness id="lingering_potion" data='{Potion:"minecraft:slowness"}'/>
-                        <lingeringPotionSlownessEx id="lingering_potion" data='{Potion:"minecraft:long_slowness"}'/>
-                    </slownessLingeringPotion>
-                    <harmingLingeringPotion>
-                        <lingeringPotionHarming id="lingering_potion" data='{Potion:"minecraft:harming"}'/>
-                        <lingeringPotionHarmingII id="lingering_potion" data='{Potion:"minecraft:strong_harming"}'/>
-                    </harmingLingeringPotion>
-                </negativeLingeringPotion>
-            </lingeringPotion>
-            <bottleOEnchanting id="experience_bottle"/>
-            <bottleEnchanting id="experience_bottle"/>
-        </potion>
-        <glassBottle id="glass_bottle"/>
-        <potionIngredient>
-            <dragonsBreath id="dragon_breath"/>
-            <netherWart id="nether_wart"/>
-            <redstone id="redstone"/>
-            <glowstoneDust id="glowstone_dust"/>
-            <sugar id="sugar"/>
-            <ghastTear id="ghast_tear"/>
-            <spiderEye id="spider_eye"/>
-            <fermentedSpiderEye id="fermented_spider_eye"/>
-            <blazePowder id="blaze_powder"/>
-            <magmaCream id="magma_cream"/>
-            <glisteringMelon id="speckled_melon"/>
-            <rabbitsFoot id="rabbit_foot"/>
-            <rabbitFoot id="rabbit_foot"/>
-        </potionIngredient>
-    </item>
-</stuff>
+	<equipment>
+		<weapon>
+			<meleeWeapon>
+				<sword>
+					<itemCreativeSword id="randomthings:creativesword"/>
```

Comment:
The default item tree should probably not explicitly name any modded items. I do some general oredict categories and such to catch common ones but there's way too many popular mods (and too many items in them) to make a heavy attempt to keep up on them.

Perhaps a secondary optional file containing the expanded entries would be more suitable?